plugins/derive/g_derive.mlg

```diff
@@ -23,6 +23,7 @@ let classify_derive_command _ = Vernacextend.(VtStartProof (Doesn'tGuaranteeOpac
 }
 
 VERNAC COMMAND EXTEND Derive CLASSIFIED BY { classify_derive_command } STATE open_proof
-| [ "Derive" identref(f) "SuchThat" constr(suchthat) "As" identref(lemma) ] ->
-  { Derive.start_deriving f.CAst.v suchthat lemma.CAst.v }
+| #[ atts = Vernacentries.DefAttributes.def_attributes; ]
```

Comment:
Isn't that the way to support attributes? When I try, say, `#[clearbody] Derive x SuchThat (x = 0) As spec.`, it says `This command does not support this attribute`.

Reply:
Vernacentries.DefAttributes.def_attributes uses the default value `NoDischarge` for `discharge` so clearbody is not allowed.

Reply:
I see. So, let's keep it as it is, until a dischargeable variant of `Derive` is possibly provided.